radix-engine-common/src/crypto/signature_validator.rs

```diff
@@ -75,36 +75,92 @@ pub fn verify_bls12381_v1(
     false
 }
 
+#[cfg(any(target_arch = "wasm32", feature = "alloc"))]
+/// Local implementation of aggregated verify for no_std and WASM32 variants (no threads)
+/// see: https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bls-signature-05#name-coreaggregateverify
+/// Inspired with blst::min_pk::Signature::aggregate_verify
+fn aggregate_verify_bls12381_v1_no_threads(
```

Comment:
Alternatively I could fork `blst` crate and modify it accordingly.
But I decided not to fork for easier maintenance in the future (eg. bumping versions).